source/_integrations/modbus.markdown

```diff
@@ -758,12 +758,11 @@ modbus:
 
 ### Services
 
-
 | Service | Description |
```

Comment:
Lets add a anchored header for each service to make it easy to link to a specific service.

Reply:
Anchors are (as far as I know) not standard in markdown, but I have added anchors using GitHub style, I hope that is correct.

Reply:
The headers will be anchors by default. See eg:

https://github.com/home-assistant/home-assistant.io/blob/56f6152f33ff66adb7ed3b53ce31e6b0a35bcc14/source/_integrations/zwave_js.markdown#L58

Reply:
Changed (nice to learn something new).